src/Sylius/Bundle/CoreBundle/Provider/DefaultTaxZoneProvider.php

```diff
@@ -20,23 +20,10 @@
 class DefaultTaxZoneProvider implements ZoneProviderInterface
```

Comment:
You got my point, we use this class in another service (hmm, there's IoC so we don't really even know if it's this one used) *as* default tax zone provider, but this implementation is based on order's channel state. We can use any other logic in `DefaultTaxZoneProvider` like returning hardcoded tax zone, getting it by address etc., so the name is not even meaningful.

`Default` is one of the keywords that are too broad and unspecific like `Manager`and so on.